src/main/java/org/web3j/gradle/plugin/GenerateContractWrappers.java

```diff
@@ -49,17 +49,17 @@ void generateContractWrappers() {
 
         final String outputDir = getOutputs().getFiles().getSingleFile().getAbsolutePath();
 
-        for (final File contractAbi : getSource()) {
+        for (final File contractBin : getSource()) {
 
-            final String contractName = contractAbi.getName().replaceAll("\\.abi", "");
+            final String contractName = contractBin.getName().replaceAll("\\.bin", "");
```

Comment:
I think you could use `String.replace()` instead of `String.replaceAll()`. `replaceAll()` compiles a regex an each call while `replace()` does not given that first argument is not real regex.